Device: cuda
Dataset: cityscapes, Train set: 2975, Val set: 500
/home/chenht/.conda/envs/epe/lib/python3.8/site-packages/torchvision/transforms/functional.py:423: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
/home/chenht/.conda/envs/epe/lib/python3.8/site-packages/torchvision/transforms/functional.py:423: UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
  warnings.warn(
[!] Retrain
Traceback (most recent call last):
  File "main.py", line 397, in get_next_boost_batch
    image, label = boost_iterator.__next__()
NameError: name 'boost_iterator' is not defined
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "main.py", line 556, in <module>
    main()
  File "main.py", line 494, in main
    boost_images, boost_labels = get_next_boost_batch()
  File "main.py", line 399, in get_next_boost_batch
    boost_iterator = boost_loader.__iter__()
NameError: name 'boost_loader' is not defined